aea/cli/packages.py

```diff
@@ -86,8 +88,20 @@ def package_manager(
     help="To sync all available packages.",
     default=False,
 )
+@click.option(
+    "-s",
+    "--source",
+    "sources",
+    type=PackagesSource(),
+    help="Provide source name from where hashes can be synced.",
+    multiple=True,
+)
```

Comment:
To sync hashes from source, provide github repo using `-s` flag

`aea packages sync -s valory-xyz/open-aea`

Before running the sync, the package manager will 

- fetch the `packages.json` from latest release for given repo
- check if the dev packages from the source repo contains the hashes for any of the third party packages
- if it contains the hash for the third party package update the hash

Then perform the packages sync

If you want to use a specific tag, provide source name as

`aea packages sync -s valory-xyz/open-aea:1.29.0`

Reply:
`valory-xyz/open-aea` should be the default.

> if it contains the hash for the third party package update the hash

What happens if it does not contain the hash? Does it log a warning?

Reply:
I don't think it makes sense to have a default since this parameter supports multiple inputs eg.. when running on a service repo it should be 

```
aea packages sync -s valory-xyz/open-aea -s valory-xyz/open-autonomy
```

No it does not log a warning if one of the sources does not contain hashes for any of the third party packages on the current repo.

Reply:
> I don't think it makes sense to have a default

Ok.

> this parameter supports multiple inputs

So which precedes which if more than one hashes exists on the given repos?

> no it does not log a warning

Maybe it should.

Reply:
On the local repo only the third party packages are updated using the dev packages on the remote repos.